### PR TITLE
IDE-2720 clean up code

### PR DESCRIPTION
**Pull request by lovettli**
Hey Greg,
This pull request just for remove duplicate code and clean up code.

### CODE REVIEW

File: tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ServiceWrapperCommand.java [hunk 1]
```diff
@@ -39,6 +39,22 @@ public ServiceWrapperCommand( IServer server )
...
+    {
+
+        if( _server == null )
+        {
+            return getStaticServiceWrapper();
+        }
+        else
+        {
+            String[] wrappers = getDynamicServiceWrapper();
+            updateServiceWrapperStaticFile( wrappers );
```
Comment (lovettli) [inline tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ServiceWrapperCommand.java]: For update static file we have another ticket to do this. So can you just review these code except this one. I will fix this issue on another ticket.

Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ServiceWrapperCommand.java]: sounds good thanks

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
Merged, thanks! See 46f62f2...bbc8e81 <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**